### PR TITLE
Stuck on firebase error

### PR DESCRIPTION
**Pull request by blunce24**
I'm working on the next checkpoint and I'm getting the following error:Firebase App named 'app' already exists. 
It seems like this error is thrown when firebase is initialized multiple times in the app. However, I only initialize firebase once, in index.html.
I'll keep working on it, but might be something to discuss during the meeting tomorrow.

### CODE REVIEW

**Comment by blunce24** [comment]
The couple things I tried weren't successful. I uploaded the changes up to this point.

File: app/scripts/app.js [hunk 1]
```diff
@@ -12,10 +24,12 @@
...
-            
     }
+    
 
+    
     angular
-        .module('blocChat', ['ui.bootstrap','ui.router', 'firebase'])
-        .config(config);
+        .module('blocChat', ['ngCookies', 'ui.bootstrap','ui.router', 'firebase'])
+        .config(config)
```
Comment (bmneely) [inline app/scripts/app.js]: .config(['$stateProvider', '$locationProvider'], config)

File: app/scripts/app.js [hunk 2]
```diff
@@ -12,10 +24,12 @@
...
+        .run(['$cookies', '$uibModal', BlocChatCookies]);
```
Comment (bmneely) [inline app/scripts/app.js]: .run(['$cookies', '$uibModal'], BlocChatCookies);

**Comment by bmneely** [comment]
Couple of suggestions from my friend.

**Pull request merged**